DESCRIPTION

```diff
@@ -30,10 +33,10 @@ URL: https://github.com/insightsengineering/rtables,
     https://insightsengineering.github.io/rtables/
 BugReports: https://github.com/insightsengineering/rtables/issues
 Depends:
-    formatters (>= 0.5.10.9004),
+    formatters (>= 0.5.11),
     magrittr (>= 1.5),
     methods,
-    R (>= 2.10)
+    R (>= 3.3)
```

Comment:
I checked the dependencies (imports and this was unreasonably low. Some imported packages like lifecycle did not exist at that time